aws-qbusiness-dataaccessor/pom.xml

```diff
@@ -0,0 +1,237 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project
+    xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>software.amazon.qbusiness.dataaccessor</groupId>
+    <artifactId>aws-qbusiness-dataaccessor-handler</artifactId>
+    <name>aws-qbusiness-dataaccessor-handler</name>
+    <version>1.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+    <properties>
+        <maven.compiler.source>17</maven.compiler.source>
+        <maven.compiler.target>17</maven.compiler.target>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <cfn.generate.args/>
+    </properties>
+
+    <dependencies>
```

Comment:
Once this is a sub module, you should be able to get rid of all the dependencies except for `aws-cloudformation-rpdk-java-plugin`